pages/api/hold/request/[id]/edd.ts

```diff
@@ -15,6 +16,12 @@ import { BASE_URL, PATHS } from "../../../../../src/config/constants"
  * Default API route handler for EDD requests
  */
 async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const patronTokenResponse = await initializePatronTokenAuth(req.cookies)
+  const cookiePatronId = patronTokenResponse.decodedPatron?.sub
+  if (!cookiePatronId) {
+    return res.status(403).json("EDD Request API - No authenticated patron")
+  }
+
```

Comment:
This asserts that a patron is logged in, but still allows one to pass the wrong patronId. We should only allow the cookied patron id to be used in the hold.